app/models/qdm/tacoma/measure.rb

```diff
@@ -0,0 +1,156 @@
+module QDM
+  # Measure contains the information necessary to represent the CQL version of a Clinical Quality Measure,
+  # As needed by the Bonnie & Cypress applications
+  class Measure
+    include Mongoid::Document
+    include Mongoid::Timestamps
+
+    DEFAULT_EFFECTIVE_DATE = Time.gm(2012, 12, 31, 23, 59).to_i
+    MP_START_DATE = Time.gm(2012, 1, 1, 0, 0).to_i
+    TYPES = %w[ep eh].freeze
+
+    # ID/other measure information
+    field :id, type: String
+    field :measure_id, type: String
+    field :hqmf_id, type: String
+    field :hqmf_set_id, type: String
+    field :hqmf_version_number, type: Integer
+    field :cms_id, type: String
+    field :title, type: String, default: ''
+    field :description, type: String, default: ''
+    field :type, type: String
+    field :category, type: String, default: 'uncategorized'
+
+    # Measure type variables
+    field :episode_of_care, type: Boolean
+    field :continuous_variable, type: Boolean
+    field :episode_ids, type: Array
+
+    # Publishing data (used by Bonnie)
+    field :published, type: Boolean
+    field :publish_date, type: Date
+    field :version, type: Integer
+
+    # ELM/CQL Measure-logic related data
+    field :elm_annotations, type: Hash
+    field :cql, type: Array
+    field :elm, type: Array
+    field :main_cql_library, type: String
+    field :cql_statement_dependencies, type: Hash
+
+    # HQMF/Tacoma-specific Measure-logic related data
+    field :population_criteria, type: Hash
+    field :data_criteria, type: Hash
+    field :source_data_criteria, type: Hash
+    field :measure_period, type: Hash
+    field :measure_attributes, type: Array
+    field :populations, type: Array
+    field :populations_cql_map, type: Hash
+    field :observations, type: Array
+
+    field :value_set_oids, type: Array, default: []
```

Comment:
These two structures should be obsolete with refactoring Value Sets to have the direct relations.